examples/isdf/00-RHF_diamond.py

```diff
@@ -0,0 +1,121 @@
+    ]
+    
+    for supercell in SuperCell_ARRAY:
+        ke_cutoff = Ke_CUTOFF[0]
+        for partition in PARTITION:   ## test different partition of atoms
+            for basis in Basis:
+                for c, rela_cutoff in zip(C_ARRAY, RELA_CUTOFF):
+                # for c in C_ARRAY:
+                    print('--------------------------------------------')
+                    print('C = %.2e, supercell = %s, kc_cutoff = %d, basis = %s, partition = %s' % (
+                        c, str(supercell), ke_cutoff, basis, partition))
+
+                    prim_cell = isdf_tools_cell.build_supercell(atm, prim_a, Ls = [1,1,1], ke_cutoff=ke_cutoff, basis=basis, pseudo="gth-pade", verbose=4)
+                    prim_mesh = prim_cell.mesh
+                    print("prim_mesh = ", prim_mesh)
+            
+                    mesh = [supercell[0] * prim_mesh[0], supercell[1] * prim_mesh[1], supercell[2] * prim_mesh[2]]
+                    mesh = np.array(mesh, dtype=np.int32)
+            
+                    cell, supercell_group = isdf_tools_cell.build_supercell_with_partition(atm, prim_a, partition=partition, Ls = supercell, ke_cutoff=ke_cutoff, mesh=mesh, basis=basis, pseudo="gth-pade", verbose=4)
+
+                    cell.incore_anyway = False
+                    cell.max_memory    = 200   # force to call with_df.get_jk
+
+                    t1 = (lib.logger.process_clock(),lib.logger.perf_counter())
+                        
+                    pbc_isdf_info = isdf_local.PBC_ISDF_Info_Quad(cell, with_robust_fitting=True, direct=False, rela_cutoff_QRCP=rela_cutoff)
+                    pbc_isdf_info.build_IP_local(c=c, group=supercell_group, Ls=[supercell[0]*4, supercell[1]*4, supercell[2]*4])
+                    print("pbc_isdf_info.naux = ", pbc_isdf_info.naux) 
+                    print("effective c = ", float(pbc_isdf_info.naux) / pbc_isdf_info.nao) 
+                    pbc_isdf_info.build_auxiliary_Coulomb()
+                                                
+                    t2 = (lib.logger.process_clock(), lib.logger.perf_counter())
+
+                    print(isdf_jk._benchmark_time(t1, t2, 'build_isdf', pbc_isdf_info))
+
+                    # for bunch_size in BUNCHSIZE_IO:
+                    ### perform scf ###
+
+                    from pyscf.pbc import scf
+
+                    t1 = (lib.logger.process_clock(), lib.logger.perf_counter())
+                    mf = scf.RHF(cell)
+                    mf.with_df = pbc_isdf_info
+                    mf.max_cycle = 32
+                    mf.conv_tol = 1e-7
+                    pbc_isdf_info.direct_scf = mf.direct_scf
+                    mf.kernel()
+                    t2 = (lib.logger.process_clock(), lib.logger.perf_counter())
+                    print(isdf_jk._benchmark_time(t1, t2, 'scf_isdf', pbc_isdf_info))
+                        
+                    del mf
+                    del pbc_isdf_info
+                exit(1)
```

Comment:
Is there a reason why you `exit(1)` in all of the examples? This signifies an error has occurred which should not be the case if it reaches the end of the code.

Reply:
Thanks for pointing out this issue. It is added just because I do not want run all the for-loops when I am testing the code. It should not appear here and I will comment this out.